src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -4,7 +4,7 @@ ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1300000;
 -- create test utility function
 -- ===================================================================
 CREATE SEQUENCE colocation_test_seq
-    MINVALUE 1
+    MINVALUE 1000
     NO CYCLE;
 /* a very simple UDF that only sets the colocation ids the same 
  * DO NOT USE THIS FUNCTION IN PRODUCTION. It manually sets colocationid column of
```

Comment:
Minot: return type of get_table_colocation_id in the test file still remains unchanged